docs/windows.md

```diff
@@ -59,7 +59,54 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (Containerd based runtimes)
+
+First install Antrea (v0.13.0+ is required for Containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will install Antrea as a Windows service. Please ensure [nssm](https://nssm.cc/)
+is on your machine, which is a handy tool to manage service on Windows. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>`
+and `<KubeletKubeconfigPath>` should be filled by you. E.g.
+```powershell
+$KubernetesVersion="v1.20.4"
+$KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
+$KubeletKubeConfig="C:/etc/kubernetes/kubelet.conf"
+```
+
+```powershell
+$TAG="v0.13.0"
+$KubernetesVersion="<KubernetesVersion>"
+$KubeConfig="<KubeconfigPath>"
+$KubeletKubeConfig="<KubeletKubeconfigPath>"
+$KubernetesHome="c:/k"
+$AntreaHome="c:/k/antrea"
+
+curl.exe -LO "https://raw.githubusercontent.com/vmware-tanzu/antrea/${TAG}/hack/windows/Helper.psm1"
+
+Import-Module ./Helper.psm1
+Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
+New-KubeProxyServiceInterface
+
+mkdir "${AntreaHome}/logs"
+nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=$KubeletKubeConfig --log-dir=c:/var/log/kube-proxy --logtostderr=false --alsologtostderr"
+nssm install antrea-agent "${KubernetesHome}/antrea/bin/antrea-agent.exe" "--config=${KubernetesHome}/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=${KubernetesHome}/antrea/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
+
+nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd
+nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
+
+Start-Service kube-proxy
+Start-Service antrea-agent
+```
+
+### Installation via wins (docker based runtimes)
+
+Installing Antrea as a wins service gives you a lot of flexibility to manage it as a Pod, but currently this only works
```

Comment:
Installing Antrea using wins..

Reply:
I just feel "as wins service" might confuse people - thinking it means Windows service.

Reply:
I think @jayunit100 is referring to https://github.com/rancher/wins. Is it correct?

Reply:
I know. But not all readers understand what is a "wins service", so I suggest to rephrase the sentence as: "Installing Antrea using wins gives.."

Reply:
Got it. I updated the sentence and added a link to the repo.

Reply:
Wins is also something else in Windows so somewhere you may need to explain this is the "Rancher" Wins and link it 
- [Rancher Wins](https://github.com/rancher/wins)
- [Microsoft Wins](https://docs.microsoft.com/en-us/windows-server/networking/technologies/wins/wins-toprl)